website/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "nomad-docs-platform",
+  "description": "Description of your website",
+  "version": "0.0.1",
+  "author": "HashiCorp",
+  "dependencies": {
+    "@bugsnag/js": "^6.5.0",
+    "@bugsnag/plugin-react": "^6.5.0",
+    "@hashicorp/nextjs-scripts": "^6.0.0-2",
+    "@hashicorp/react-alert": "^2.0.0",
+    "@hashicorp/react-button": "^2.1.3",
+    "@hashicorp/react-case-study-slider": "^2.0.4",
+    "@hashicorp/react-consent-manager": "^2.0.3",
+    "@hashicorp/react-content": "^2.2.0",
+    "@hashicorp/react-docs-sidenav": "^3.0.2",
+    "@hashicorp/react-docs-sitemap": "^1.0.0",
+    "@hashicorp/react-footer": "^3.1.6",
+    "@hashicorp/react-global-styles": "^4.0.8",
+    "@hashicorp/react-hero": "^3.0.0",
+    "@hashicorp/react-image": "^2.0.1",
+    "@hashicorp/react-inline-svg": "^1.0.0",
+    "@hashicorp/react-mega-nav": "^4.0.0-0",
```

Comment:
Minor nit: (@treehouseops will thank you!) Can this get bumped to `4.0.1-2` (same as Vault) so we grab the update to "HashiCorp Stack" (rather than Suite).

Reply:
Good catch, updated!